digitalocean/kubernetes/resource_kubernetes_cluster.go

```diff
@@ -294,6 +301,18 @@ func resourceDigitalOceanKubernetesClusterCreate(ctx context.Context, d *schema.
 		return diag.Errorf("Error creating Kubernetes cluster: %s", err)
 	}
 
+	if d.Get("registry_integration") == true {
+		err = enableRegistryIntegration(client, cluster.ID)
+		if err != nil {
+			return diag.Errorf("Error enabling registry integration: %s", err)
+		}
+	} else {
+		err = disableRegistryIntegration(client, cluster.ID)
+		if err != nil {
+			return diag.Errorf("Error disabling registry integration: %s", err)
+		}
+	}
```

Comment:
In the create method, I think we can remove this else statement. As the registry integration is not enabled, this adds an unneeded API call.

Reply:
Thanks for the note, I changed the code now